src/main/java/bio/terra/service/filedata/google/firestore/FireStoreBatchQueryIterator.java

```diff
@@ -13,9 +13,12 @@ public class FireStoreBatchQueryIterator {
   private final Query baseQuery;
   private final int batchSize;
   private List<QueryDocumentSnapshot> currentList;
-  private int count;
-  private FireStoreUtils fireStoreUtils;
-  private Firestore firestore;
+  private int count = 0;
+  private int totalSize = 0;
```

Comment:
This is OK but redundant as `0` is the default value for an uninitialized class field.
```suggestion
  private int count;
  private int totalSize;
```

Reply:
Oh I see what you mean now!